pkgs/build-support/php/build-php.nix

```
@@ -0,0 +1,145 @@
+{ stdenvNoCC, lib, writeTextDir, php, unzip, xz, git, makeBinaryWrapper, jq }@inputs:
+
+let
+  buildPhpProjectOverride = finalAttrs: previousAttrs:
+
+  let
+    composerLock = finalAttrs.composerLock or null;
+    php = finalAttrs.php or inputs.php;
+    composer = finalAttrs.composer or inputs.php.packages.composer;
```

```suggestion
{ stdenvNoCC, lib, writeTextDir, php, unzip, xz, git, makeBinaryWrapper, jq }:

let
  buildPhpProjectOverride = finalAttrs: previousAttrs:

  let
    composerLock = finalAttrs.composerLock or null;
    php = finalAttrs.php or php;
    composer = finalAttrs.composer or php.packages.composer;
```
same as in the other comment. I think we only really use @inputs (or more commonly @args) we we are capturing inputs via ``...``.